### ISSUE TITLE
Add information from 'synonyms' section

### ISSUE THREAD

**Issue by radomirbosak** [radomirbosak/duden-down#12]
For example for 'Petersilie', this would be something like:
```
>>> petersilie.synonyms
['(schweizerisch) Peterli', '(bayrisch, österreichisch umgangssprachlich) Petersil', '(südwestdeutsch und schweizerisch mundartlich) Peterle']
```

**Comment by radomirbosak** [comment]
Fixed in [develop](https://github.com/radomirbosak/duden-down/tree/develop) branch

**Comment by radomirbosak** [comment]
Fixed in [0.5.0](https://github.com/radomirbosak/duden-down/releases/tag/v0.5.0)
